gp_emulator/GaussianProcess.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import scipy.spatial.distance as dist
 import random
+import sys
+#import _gpu_predict
 
 def k_fold_cross_validation(X, K, randomise = False):
```

Comment:
Where clients are using bad programming practices like single-letter variable names, feel free to modify their code. E.g. here X, K can be data, subsamples_count